### PR TITLE
fixed demo for m-button with react

### PR DESCRIPTION
**Pull request by zuzana**
### Why?

Fixed demo for toggling state of button. 
I don't know exactly why wrapping text in `<span>` or any other component works. 

### How to test?

1. Prepare
        
        rm -rf node_modules
        npm i 
        npm run serve

1. Navigate to [http://localhost:3000/#react](http://localhost:3000/#react)
1. Try toggling button

### Issue with babel plugin external-helpers

 I had issue with `external-helpers` plugin. I solved it according [this comment](https://github.com/storybooks/storybook/issues/1320#issuecomment-310777396)

      Web Component AXA-BUTTON has an error while loading its template:
      ReferenceError: babelHelpers is not defined
      
      Stack Trace: ReferenceError: babelHelpers is not defined
          at template (http://localhost:3000/components/m-button/index.js:857:74)
          at HTMLElement.render (http://localhost:3000/components/m-button/index.js:1624:23)
          at HTMLElement.connectedCallback (http://localhost:3000/components/m-button/index.js:1553:12)
          at http://localhost:3000/components/m-button/index.js:1850:25
          at HTMLDocument.once (http://localhost:3000/components/m-button/index.js:1827:12)

### CODE REVIEW

**Comment by AndyOGo** [comment]
@zuzana 
Thanks for your contribution.
Sorry we haven't mentioned it anywhere, though may I ask you to not include distribution/generated code into the PR (`dist/` and `docs/` folder).
Those stuff is only necessary for packaging and gets automatically added by every release by running `npm run release`

File: src/demos/demo.react.jsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ class MyEventDemoReact extends React.Component {
   render() {
     return (
       <axa-button onClick={this.handleClick}>
-        {this.state.isToggleOn ? 'ON' : 'OFF'}
+        <span>{this.state.isToggleOn ? 'ON' : 'OFF'}</span>
```
Comment (AndyOGo) [inline src/demos/demo.react.jsx]: @zuzana 
The real issue is that we don't support incremental rendering of any web component right now.
Adding `<span>` isn't the proper way to deal with it. All necessary features are specified here #147. And for easy testing support of interactive demos #183

Comment (AndyOGo) [reply src/demos/demo.react.jsx]: Correct place to fix all of those is:
https://github.com/axa-ch/patterns-library/blob/master/src/components/_abstract/component-types.js

Comment (LucaMele) [reply src/demos/demo.react.jsx]: Agree for when it goes production ready. As we are still beta, this quick fix will enhance the demo and therefore for me is ok to merge. The dist folder however should be reset and we will make a deploy as soon as this is merged in order to update the dist and docs folder.

Comment (zuzana) [reply src/demos/demo.react.jsx]: changes to dist folder reverted.

**Comment by LucaMele** [comment]
@zuzana, you can merge

**Pull request merged**